lucet-runtime/lucet-runtime-tests/src/timeout.rs

```diff
@@ -252,25 +269,42 @@ macro_rules! timeout_tests {
 
             let kill_switch = inst.kill_switch();
 
-            let t = thread::spawn(move || {
-                assert!(kill_switch.terminate().is_err()); // fails too soon
-                thread::sleep(Duration::from_millis(100));
-                assert!(kill_switch.terminate().is_ok()); // works
-                thread::sleep(Duration::from_millis(100));
-                assert!(kill_switch.terminate().is_err()); // fails too soon
-            });
+            let helper = thread::Builder::new()
+                .name("helper".to_owned())
+                .spawn(move || {
+                    // This fires first, and will work properly, terminating the instance.
+                    assert_eq!(kill_switch.terminate(), Ok(KillSuccess::Cancelled));
+                    // Next, we will delay for 100ms and try to terminate the instance again.
+                    // This should fail, because we have already terminated the instance.
+                    thread::sleep(Duration::from_millis(100));
+                    assert_eq!(kill_switch.terminate(), Err(KillError::NotTerminable));
```

Comment:
specifying the `KillError` and `KillSuccess` variants for the sake of consistency.